docs/contribution_guide.md

```diff
@@ -58,6 +58,7 @@ We strive to maintain a friendly and inclusive community. We have not yet establ
     export BUILD_ID=""
     export JOB_NAME_SAFE=""
     export JOB_NAME=""
+    export PULL_NUMBER=""
```

Comment:
Can you add an optional value to the `development/env.list` file as well? You can put it near `BUILD_ID` and comment it out.